src/PostComment.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import { getUsers, postComment } from "./apis"
+
+
+export const PostComment = ({setListOfComments}) => {
+    const [isAddedToCommentList, setIsAddedToCommentList] = useState(false)
+    const [users, setUsers] = useState([])
+    const {review_id} = useParams()
+
+    useEffect(()=> {
+        getUsers().then((users) => {
+          const usernames = users.map((user) => {return user.username})
+            setUsers(usernames)
+        })
```

Comment:
OK this is interesting - getting all the users first.

Reply:
If you do the comment posting as a hard coded user, then this useeffect may not be needed here any longer.

React Context, covered tomorrow, will help with this functionality later on :)